### PR TITLE
about-done

### PR DESCRIPTION
**Pull request by vladBrownDev**

### CODE REVIEW

File: src/components/about/About.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
+p{ 
+    margin:0;
```
Comment (stasoz1) [inline src/components/about/About.css:2]: Don't forget about space after :. It's related for all rows in this file (11, 42, 43, etc)
e.g `margin: 0`.

File: src/components/about/About.jsx [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
 const About = () => {
     return (
-        <div>About</div>
+        <section id="about">
+            <div id="aboutLeft">
+                <h1 className="mainHeading">
+                    Обо мне
+                </h1>
+                <div id="aboutDesc">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
```
Comment (stasoz1) [inline src/components/about/About.jsx:13]: For the future we can create separate file with constant and export them here.

E.g
constants.js will contain this 
`export const mockText = 'Lorem ipsum...'`
then in About.jsx 
`import { mockText } from 'constants.js'`
and insert in 'aboutDesc' section
```
<div id="aboutDesc">
    {mockText }
</div>
```

Thus, we will have a separate file with easily changeable text

Comment (stasoz1) [reply src/components/about/About.jsx:13]: @vladBrownDev what do you think about it?

**Comment by vladBrownDev** [comment]
ыыыыы

**Review by stasoz1 — CHANGES_REQUESTED** [review]

**Pull request merged**